colosseum/declaration.py

```diff
@@ -161,10 +170,14 @@ def __init__(self, **style):
     border_width = directional_property('border%s_width', initial=0)
 
     # 8.5.2 Border color
-    border_top_color = validated_property('border_top_color', choices=BORDER_COLOR_CHOICES, initial=TRANSPARENT)
-    border_right_color = validated_property('border_right_color', choices=BORDER_COLOR_CHOICES, initial=TRANSPARENT)
-    border_bottom_color = validated_property('border_bottom_color', choices=BORDER_COLOR_CHOICES, initial=TRANSPARENT)
-    border_left_color = validated_property('border_left_color', choices=BORDER_COLOR_CHOICES, initial=TRANSPARENT)
+    border_top_color = validated_property('border_top_color', choices=BORDER_COLOR_CHOICES,
+                                          initial=OtherProperty('color'))
```

Comment:
Nice! It's all starting to come together! :-)

Reply:
:-)